[Task] Description about how to find extensions and renaming management (#535)
TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted: Documentation/Extensions/HowToFindExtensions.rst

.. include:: /Includes.rst.txt

.. _how-to-find-extensions:

======================
How to find extensions
======================

For beginners, finding the right extensions can enhance their TYPO3 experience
significantly. Here is a brief guide on how to search for extensions and where
to find the necessary information.

#.  `TYPO3 Extension Repository (TER) <https://extensions.typo3.org/>`__

>     The TYPO3 Extension Repository (TER) is online platform, that hosts

Reviewer: ```suggestion
    The TYPO3 Extension Repository (TER) is an online platform, that hosts
```

    thousands of extensions created by the TYPO3 Community.
    Here you can:

    *   Search by keyword
    *   Explore categories
    *   Filter results

#.  `Packagist <https://packagist.org//>`__

    Packagist serves as the default package repository for PHP packages using Composer,
    including TYPO3 extensions. You can search for TYPO3 extensions on Packagist
    by entering relevant keywords. Many TYPO3 extensions are available via Composer,
    which allows for easier dependency management and installation. This is
    particularly useful for developers looking to integrate extensions into

>     their projects.

Reviewer: You can also filter by TYPO3 extensions:

![image](https://github.com/user-attachments/assets/348ef42f-af47-413d-a201-5e269b9a8fd8)

https://packagist.org/?type=typo3-cms-extension

..  _tips-for-choosing-extensions:

> Tips for Choosing the right TYPO3 Extension

Reviewer: ```suggestion
Tips for choosing the right TYPO3 extension
```

We use title case in the docs.

    incompatible extension can lead to errors or unexpected issues.

*   Popularity and reviews:

    Extensions that are frequently used and have good reviews are often more
    reliable. Look at download numbers, ratings, and user feedback in the TER
    or on GitHub to get a sense of the extension’s quality.

*   Support and updates:

    Check if the extension is actively maintained and updated to work with the
    latest TYPO3 versions. An extension with recent updates is more likely to
    be secure and compatible with modern TYPO3 standards.

*   Documentation:

    Good documentation is essential, especially for beginners. Ensure that the
    extension has clear setup and configuration guides, either in the TER, on
    the developer's website, or on GitHub.

*   Performance and security:

    Extensions can affect your site’s performance, so choose extensions that
    are optimized and well-coded. Additionally, check for any reported security
    vulnerabilities, particularly for older extensions, to keep your site secure.

*   Customizability and flexibility:

    If you have specific needs, ensure that the extension is flexible enough to

>     be customized or configured as required. Some extensions provide hooks or

Reviewer: ```suggestion
    be customized or configured as required. Some extensions provide PSR-14 events or
```

As hooks are a dying species, we should avoid them in new docs.
